recipe/conda_build_config.yaml

```diff
@@ -0,0 +1,3 @@
+# Please consult conda-forge/core before doing this
+MACOSX_SDK_VERSION:        # [osx and x86_64]
+  - "10.12"                # [osx and x86_64]
```

Comment:
@conda-forge/core upstream introduced os/log.h in their code which is only available in 10.12 and above (if I understand correctly... maybe 10.11 too). Consulting you on this --- I will leave this up for a week, if you see it, please merge; if not, I will merge in a week or so.

Reply:
+1

Reply:
I will take that as a vote confidence, merging